### PR TITLE
Better Variable name for its function

### PR DESCRIPTION
**Pull request by wangzhen11aaa**
Does new name "DuplicatedBackup" reflect the meaning of the function than "Required"?

### CODE REVIEW

File: pkg/metadata/metadata.go [hunk 1]
```diff
@@ -51,9 +51,9 @@ type TableMetadata struct {
 }
 
 type Part struct {
-	Partition string `json:"partition,omitempty"`
-	Name      string `json:"name"`
-	Required  bool   `json:"required,omitempty"`
+	Partition      string `json:"partition,omitempty"`
+	Name           string `json:"name"`
+	DuplicatedPart bool   `json:"required,omitempty"`
```
Comment (Slach) [inline pkg/metadata/metadata.go:56]: Sorry, I can't approve this PR
I undestand maybe  "requiered" is not so mental name as you expected in the code, but differences between `json` and internal variable name is worst variant.

**Review by Slach — COMMENTED** [review]
Sorry, I can't approve this PR
I undestand maybe "requiered" is not so mental name as you expected in the code, but differences between json and internal variable name is worst variant.

File: pkg/metadata/metadata.go (hunk 1 above)
Comment (wangzhen11aaa) [reply pkg/metadata/metadata.go:56]: > Sorry, I can't approve this PR I undestand maybe "requiered" is not so mental name as you expected in the code, but differences between `json` and internal variable name is worst variant.

What does this "required" mean?

Comment (Slach) [reply pkg/metadata/metadata.go:56]: For incremental backup, we have the parent backup which marked as `required` in backup_name/metadata.json
and some parts are not present in incremental backup and present only in `required` backup, so it also marked as `required`

During download incremental backup, `required` backup downloads first and during unpack `required` parts create as hard links.

**Pull request closed without merging**